### PR TITLE
Repro/auto dashboards

### PR DESCRIPTION
**Pull request by Opalevanescence**
Test to make sure the x-rays load (from issue #12917)

### CODE REVIEW

File: frontend/test/metabase/scenarios/dashboard/x-rays.cy.spec.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    cy.contains("A look at your People table");
+    cy.contains("A look at your Orders table");
+    cy.contains("A look at your Products table");
+    cy.contains("A look at your Reviews table");
+  });
+
+  it.skip("should be populated (Issue #12917)", () => {
+    cy.visit("/");
+    cy.findByText("People table").click();
+    cy.findByText("Something's gone wrong").should("not.exist");
```
Comment (paulrosenzweig) [inline frontend/test/metabase/scenarios/dashboard/x-rays.cy.spec.js:18]: I'm guessing this test was written while it was still broken. Now that it's fixed, I think it'd be better to assert what should be there.

Comment (Opalevanescence) [reply frontend/test/metabase/scenarios/dashboard/x-rays.cy.spec.js:18]: @paulrosenzweig I don't think this issue is fixed yet.

Comment (paulrosenzweig) [reply frontend/test/metabase/scenarios/dashboard/x-rays.cy.spec.js:18]: @Opalevanescence It seems to be working for me. Could you be running an old build of the frontend? We should jump on that if it's still broken.

Comment (paulrosenzweig) [reply frontend/test/metabase/scenarios/dashboard/x-rays.cy.spec.js:18]: Ah sorry I think this was branched off master before the fix. Merging master or release-0.36.x should pull in the fix. While doing that, it's probably best to switch the base branch to release-0.36.x.

Comment (Opalevanescence) [reply frontend/test/metabase/scenarios/dashboard/x-rays.cy.spec.js:18]: I tried rebasing (and pulling in release-0.36.x) and rebuilt the frontend, but it still isn't working. Also, if this issue is fixed, why is #12917 still open?

Comment (paulrosenzweig) [reply frontend/test/metabase/scenarios/dashboard/x-rays.cy.spec.js:18]: Hmm I'm not sure what's happening then. I just checked out this branch, merged in `release-0.36.x`, and x-rays are working for me. Is it the same root cause as before? Will a new dashboard with just a text card fail in the same way?

#12917 is an issue to create a test. The issue for the bug is #12873.

Comment (paulrosenzweig) [reply frontend/test/metabase/scenarios/dashboard/x-rays.cy.spec.js:18]: Maybe it's the issue where you need to restart cypress to get the newly built frontend in the uberjar running tests? That wouldn't explain seeing the issue outside of cypress though if you're seeing that.

Comment (Opalevanescence) [reply frontend/test/metabase/scenarios/dashboard/x-rays.cy.spec.js:19]: Thanks for helping with getting everything running on my computer! This page is showing up correctly now.

**Review by paulrosenzweig — APPROVED** [review]

**Pull request merged**